sdk/keyvault/keyvault-certificates/review/keyvault-certificates.api.md

```diff
@@ -345,6 +351,9 @@ export interface KVPollerLike<TState extends PollOperationState<TResult>, TResul
     toString(): string;
 }
 
+// @public
+export const LATEST_STABLE_KEYVAULT_SERVICE_API_VERSION = "7.1";
```

Comment:
This will still be "7.0".

Reply:
Do we need to export this const, even?

Reply:
Not really, I'll keep it unexported.

Reply:
I believe that Heath said that this version would be 7.1-preview in this preview version. I'll use that for now!